server/sonar-server/src/main/java/org/sonar/server/computation/dbcleaner/ProjectCleaner.java

```diff
@@ -53,11 +54,11 @@ public ProjectCleaner(PurgeDao purgeDao, DefaultPeriodCleaner periodCleaner, Pur
     this.purgeListener = purgeListener;
   }
 
-  public ProjectCleaner purge(DbSession session, IdUuidPair idUuidPair, Settings projectSettings) {
+  public ProjectCleaner purge(DbSession session, IdUuidPair idUuidPair, Settings projectSettings, Collection<String> disabledComponentUuids) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)